operators/config/e2e/rbac.yaml

```diff
@@ -40,6 +40,7 @@ rules:
       - watch
       - delete
       - create
+      - update
```

Comment:
Why do we need this now?

Reply:
When I ran the e2e tests one failed because the e2e service account could not update a secret.
I don't known the exact reason, given the fact that this SA is just used for e2e tests I just granted the authorization.

Reply:
The keystore tests we re-enabled do create and update a secret.